### ISSUE TITLE
Create Introduction Screens

### ISSUE THREAD

**Issue by chhavip** [systers/powerup-android#31]
An introductory splash screen to give user familiarity with the game would be beneficial as is the norm with any gaming application.

**Comment by alma-castant** [comment]
This should only be played the first time the app is open, and should explain what the user has to do in the game with images and screenshots

**Comment by medhach** [comment]
[IntroScreens.pdf](https://github.com/systers/powerup-android/files/59529/IntroScreens.pdf)
I have made these screens as the introduction screens. These could be shown the first time the app is opened by the user.
I have tried to keep the content simple and small, but do let me know if I should change it. :)

**Comment by alma-castant** [comment]
Looks really good @medhach 
The only suggestions I would have are:
- I wouldn't write the text "swipe". You can either have a button saying "next" that lets the user move forward when clicking on it or use swipe dots like [these](http://www.appcoda.com/wp-content/uploads/2013/06/pageviewcontroller1.png) to let the user know there is something else when they swipe

- In the last screen, you can probability make the text shorter so it's quicker to read. Something like "You will earn points as you make decisions in the game. Points can be used to purchase...". The wording of self-steem and understanding of reproductive health may be too complicated for 12-year-olds

Other than that, I think your design is very good and I would go ahead implementing it :)

**Comment by medhach** [comment]
Will make the changes and pull a request soon, Thank you!

**Comment by chhavip** [comment]
@medhach Are you working on this issue? Would love to see the PR :)

**Comment by medhach** [comment]
I have included the introduction screens in my proposal. Will submit it
soon. :)

**Comment by aanchal07** [comment]
Has it been done?
I would like to work on this issue if not.

**Comment by chhavip** [comment]
@medhach Will you be submitting the PR?

**Comment by medhach** [comment]
@aanchal07 You can go ahead.:)

**Comment by aanchal07** [comment]
@medhach Great! :)

**Comment by pandilladivya** [comment]
@aanchal07 , @chhavip Has this been done?

If not, I would like to work on it ..

Thank You!

**Comment by BaaniLeen** [comment]
![sample](https://cloud.githubusercontent.com/assets/22029744/24079663/f90a545a-0cb2-11e7-8b16-af1492852d15.gif)

We can have an introduction to the app(similar to the above GIF) after the splash screen which gives the user information about the aim of the project and the instructions like "You will earn points as you make decisions in the game" as suggested by @alma-castant ...
I can implement this using the Material Intro Tutorial Library.

@chhavip  Can I work on this? Thanks!

**Comment by codingblazer** [comment]
@BaaniLeen Its been already solved via PR #323 :-)

**Comment by BaaniLeen** [comment]
@codingblazer Okay thanks..! :-)
perhaps I missed it since the PR has not been merged yet.

**Comment by codingblazer** [comment]
@aanchal07 This already has a PR #323 . Does this require any changes ?

**Comment by aanchal07** [comment]
@codingblazer As we will be redesigning the intro screens, I have closed that PR for now. I will put this issue on hold.

**Comment by codingblazer** [comment]
Yeah sure !!

**Comment by geekanamika** [comment]
Is anybody working on this issue ? I would like to work on it :)

**Comment by codingblazer** [comment]
It's not required as of now. Not until we have more designs from the design team on this. Introduction screens for mini-games are already done.

**Comment by geekanamika** [comment]
@codingblazer @MeepyMay Were new designs created during Outreachy program ? If this issue is still open to work , I would like to work on it :)

**Comment by Rimjhim28** [comment]
Closing the PR for now. Can be reopened after updating the issue according to template and discussion with the community
